extensions/quartz/deployment/src/main/java/io/quarkus/quartz/deployment/QuartzProcessor.java

```diff
@@ -0,0 +1,82 @@
+package io.quarkus.quartz.deployment;
+
+import static io.quarkus.deployment.annotations.ExecutionTime.RUNTIME_INIT;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.quartz.simpl.CascadingClassLoadHelper;
+import org.quartz.simpl.RAMJobStore;
+import org.quartz.simpl.SimpleThreadPool;
+
+import io.quarkus.arc.deployment.AdditionalBeanBuildItem;
+import io.quarkus.arc.deployment.BeanContainerBuildItem;
+import io.quarkus.deployment.Capabilities;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.annotations.Record;
+import io.quarkus.deployment.builditem.CapabilityBuildItem;
+import io.quarkus.deployment.builditem.ServiceStartBuildItem;
+import io.quarkus.deployment.builditem.nativeimage.ReflectiveClassBuildItem;
+import io.quarkus.deployment.logging.LogCleanupFilterBuildItem;
+import io.quarkus.quartz.runtime.QuartzRecorder;
+import io.quarkus.quartz.runtime.QuartzRuntimeConfig;
+import io.quarkus.quartz.runtime.QuartzScheduler;
+import io.quarkus.quartz.runtime.QuartzSupport;
+
+/**
+ * @author Martin Kouba
+ */
+public class QuartzProcessor {
```

Comment:
Maybe we can produce a `quartz` FeatureBuildItem in this class?

Reply:
Okay, makes totally sense. Sorry, I had went ahead and added it in my PR . I’ll remove it once this is merged. Thanks.

Reply:
No problem ;-).